integration_test.go

```diff
@@ -43,10 +43,52 @@ func TestCreateAWSIntegration(t *testing.T) {
 	autopilot.Equals(t, "AWS - XXXX", result.Name)
 }
 
+func TestCreateAzureResourcesIntegration(t *testing.T) {
+	// Arrange
+	testRequest := autopilot.NewTestRequest(
+		`mutation AzureResourcesIntegrationCreate($input:AzureResourcesIntegrationInput!){azureResourcesIntegrationCreate(input: $input){integration{id,name,type,createdAt,installedAt,... on AwsIntegration{iamRole,externalId,awsTagsOverrideOwnership,ownershipTagKeys},... on AzureResourcesIntegration{tenantId,subscriptionId,lastSyncedAt,aliases},... on NewRelicIntegration{baseUrl,accountKey}},errors{message,path}}}`,
+		`{"input": { "name": "new azure resources", "tenantId": "12345678-1234-1234-1234-123456789abc", "subscriptionId": "12345678-1234-1234-1234-123456789def", "clientId": "XXX_client_id_XXX", "clientSecret": "XXX_client_secret_XXX"}}`,
+		`{"data": {
+      "azureResourcesIntegrationCreate": {
+        "integration": {
+          {{ template "id1" }},
+          "name": "new azure resources",
+          "type": "azureResources",
+          "createdAt": "2024-07-04T16:25:29.574450Z",
+          "installedAt": "2024-07-04T16:25:28.541124Z",
```

Comment:
these are here but not in the `AzureResourcesIntegrationFragment` ?

Reply:
`createdAt` and `installedAt` are fields in the [Integration struct](https://github.com/OpsLevel/opslevel-go/pull/443/files#diff-b7b79f1ac09ee75185e8a967df69488f9fe9597d89aaa80d50296d9c028ee7d9R60-R61)